docroot/themes/humsci/humsci_basic/src/scss/components/_card.scss

```diff
@@ -249,14 +251,34 @@
     align-items: center;
     width: 100%; // large area to click
 
+    .views-field {
+      width: 100%;
+    }
+
+    a {
+      display: block;
+      color: transparent; // `hide` the text but make sure screen readers can still read it
+      height: hb-calculate-rems(44px);
+      width: 100%;
+      margin-left: hb-calculate-rems(-44px);
+      text-decoration: none;
+      z-index: $hb-z-index-decorative-link;
```

Comment:
hm why does it need zindex? can you leave a comment?

Reply:
I'm glad you made this request. I found that clicking on the arrow icon in the horizontal card wasn't working and that I needed to add this to another class as well. :tada: